tests/upgrades/test_satellite_maintain.py

```diff
@@ -139,16 +75,7 @@ def test_post_satellite_maintain_upgrade_list_versions(self, target_sat):
         :expectedresults: Versions should be next z-stream.
 
         """
-        (
-            satellite_version,
-            upgradable_version,
-            major_version_change,
-            y_version,
-        ) = self.satellite_upgradable_version_list(target_sat)
-        if satellite_version:
-            zstream_version, next_version = self.version_details(
-                satellite_version[0], major_version_change, y_version, upgrade_stage="post-upgrade"
-            )
-        else:
-            next_version = -1
-        assert next_version in upgradable_version
+        zstream = ".".join(target_sat.version.split('.')[0:2]) + ".z"
```

Comment:
Use a single quote or double quote to maintain consistency

Reply:
Amended. Thanks.